app/models/intra_alaska_base_linehaul.rb

```diff
@@ -0,0 +1,7 @@
+class IntraAlaskaBaseLinehaul < ApplicationRecord
```

Comment:
I had shied away from this, despite having used STI for offices in an earlier incarnation of the locator maps. I wanted to keep the data separate because the calculations are somewhat different, but in retrospect that seems a bit silly.

Reply:
Good news! We're still using STI for offices. 😬

Reply:
I tried STI, and it doesn't seem to play nice with activerecord-import (the .import method isn't getting added to the child model classes). There may or may not be an easy solution to this.